js/everything.js

```diff
@@ -368,13 +380,16 @@ function removeSearchQueryFromURL(url) {
     }
     return url;
 }
-
-function makeListenURL(videoID) {
+function makeListenURL(videoID,videoPosition) {
     var url = removeSearchQueryFromURL(window.location.href);
     // Remove any #s which break functionality
     url = url.replace("#", "");
-
-    return url + "?v=" + videoID;
+    if (videoPosition) {
```

Comment:
Let's clean this up to have a single `return`. The `videoPosition` param should just be a `Number` not the full `&t=x` string